hugo.yaml

```diff
@@ -15,6 +15,10 @@ languages:
     languageName: English
     params:
       description: The OpenTelemetry Project Site
+  ja:
+    languageName: 日本語 (Japanese)
+    # params:
+    #   description:
```

Comment:
We'll need a Japanese translation of "The OpenTelemetry Project Site" to put in these lines. No rush, it can be handled in a followup PR.